site/docs/components/loading-skeleton.mdx

```diff
@@ -6,7 +6,7 @@ tags: ["Skeleton", "Stencil", "Loading Placeholders", "Placeholder UI"]
 githubLabels: ["component:loading-skeleton"]
 needToKnow:
 - Give people an idea of what’s about to come and what’s happening (it's currently loading) when a page full of content/data takes more than 300ms to load.
-demoStoryId: components-loading-skeleton-loading-skeleton--sticker-sheet-default
+demoStoryId: components-loading-skeleton-loading-headings--sticker-sheet-reversed
```

Comment:
Not sure what would be the most optimal example, but perhaps the site docs should embed this story instead? 
https://dev.cultureamp.design/KDS-47/deprecate-loading-placeholder/storybook/?path=/story/components-loading-skeleton--sticker-sheet-default

Reply:
It unfortunately isn't responsive and breaks - we would want to make a separate one instead maybe?

Reply:
Fair enough. A separate story later which showcases the different loading skeletons might be nice just for the site (without the sample loaded content).